docs/map.json

```diff
@@ -1578,16 +1583,6 @@
               "name": "Advanced Configuration",
               "id": "advanced-config",
               "file": "nx-cloud/private/advanced-config"
-            },
-            {
-              "name": "Kubernetes Setup",
-              "id": "kubernetes-setup",
-              "file": "nx-cloud/private/kubernetes-setup"
-            },
-            {
-              "name": "Standalone",
-              "id": "standalone",
-              "file": "nx-cloud/private/standalone"
```

Comment:
we need to add redirect rules for these two pages that are deleted.  I'm assuming they should redirect to the new page you made.

Reply:
ah good catch - didn't realise we can add redirects 👍 

should be good now I think!